### PR TITLE
update fields to retrieve HostID and name

### PR DESCRIPTION
**Pull request by Freedisch**
**Description**

This PR fixes Update fields for Models, Relationships, and components to display host Id and name 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@zakisk

File: models/meshmodel/core/v1alpha1/models.go [hunk 1]
```diff
@@ -38,6 +39,9 @@ type Model struct {
 	Version     string                 `json:"version"`
 	DisplayName string                 `json:"displayName" gorm:"modelDisplayName"`
 	Category    Category               `json:"category"`
+	HostID	    uuid.UUID			   `json:"hostID"`
```
Comment (zakisk) [inline models/meshmodel/core/v1alpha1/models.go:42]: @Freedisch please add linting at the moment you can see that stylistically code isn't good.

**Review by zakisk — CHANGES_REQUESTED** [review]
@Freedisch code in terms of solving issue is good but linting must be added

**Review by zakisk — APPROVED** [review]

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
@Freedisch Have you verified your changes? Do you get `HostID, HostName, DisplayHostName` when you query for model? 
You also need to modify GetModel IMO.

**Comment by Freedisch** [comment]
@MUzairS15 ,  I already modified GetModel it's in another PR in Meshery, 
here the body response
image

**Comment by MUzairS15** [comment]
> @MUzairS15 , I already modified GetModel it's in another in this PR [#8257](https://github.com/meshery/meshery/pull/8257), here the body response image
> 
> Regarding the model, are the available models registered under a host/or registrant? image

This is what I was referring to the second screenshot the hostname and id are empty, you need to modify [getmodels](https://github.com/meshery/meshkit/pull/331)

**Comment by Freedisch** [comment]
@MUzairS15 
image

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
@Freedisch I understand that you are retrieving registrants and assigning host details (in your meshery PR), but host details are now included in Model constructs so why not set host details while retrieving models? https://github.com/meshery/meshkit/pull/332/files#diff-69b1a3b3507297713ffe3c635a25bbf8a8807b7a2c316a8bed048c222fde7b55R194

**Comment by leecalcote** [comment]
merge conflict

**Comment by Freedisch** [comment]
@MUzairS15, thoughts? i updated the pr

**Comment by leecalcote** [comment]
Rady for re-review, @MUzairS15

**Comment by Freedisch** [comment]
@leecalcote, sorry forgot to close this one. The changes here have been already merged here #337

**Pull request closed without merging**